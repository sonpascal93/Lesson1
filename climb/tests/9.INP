500
-39 -24 -31 19 -10 20 45 -37 -29 23 16 -11 11 20 41 21 -40 -30 4 14 13 -35 -18 0 10 30 6 47 7 18 -12 -24 46 49 -27 -20 -38 37 47 9 -33 26 45 5 -29 -30 22 -38 -9 0 40 -7 -49 9 -14 0 46 0 -43 2 -1 -48 3 -7 -40 47 9 6 11 33 -36 12 31 -40 31 -16 -17 23 41 -40 -28 -16 -46 -10 -12 32 1 0 -26 28 11 -4 -36 1 -25 -36 7 28 5 -23 -24 25 1 31 20 41 3 -3 43 -45 14 -10 41 -32 -42 28 38 -26 28 47 6 -48 48 -7 6 -2 -20 -35 35 -4 -31 -2 0 30 -45 4 15 18 -22 -43 49 40 -9 -31 -27 -32 -36 1 25 -20 -3 -19 -44 -25 -4 -27 -43 36 -15 -10 11 -35 3 -41 33 -14 38 -35 -16 -27 -12 -12 22 8 25 17 47 38 -35 35 -28 38 -40 -42 -16 -24 -32 -1 28 41 -32 37 36 16 33 -4 33 13 3 27 21 -8 35 -2 -16 44 -46 -2 -29 -18 9 35 44 -38 20 49 -26 -41 -17 -12 24 39 -31 -23 14 -24 -21 4 9 -26 8 -1 13 -37 -20 -25 5 43 -5 37 17 40 34 -6 -27 32 -19 -33 -16 -10 -43 46 25 1 21 10 -16 3 -30 40 -20 38 9 9 6 18 -24 24 -44 3 34 -19 40 42 -31 38 -26 24 20 -4 -27 -46 15 -37 31 -16 -25 21 26 32 -8 -25 -42 15 11 -23 -35 25 43 -42 38 -2 35 -48 26 -25 -7 -19 44 -10 48 11 31 14 -34 22 5 4 1 -15 -48 3 38 -38 17 7 -33 -46 -30 -49 35 21 -41 21 -24 8 20 -26 18 39 47 -31 35 31 20 -29 0 -16 -9 -19 -27 -32 4 46 49 46 -22 49 30 -30 -42 44 -47 6 -41 -23 -17 46 -49 -45 36 37 -35 -45 25 8 -35 4 -1 -5 7 18 -8 -46 26 37 -23 44 -16 8 -38 -48 0 21 -4 6 -29 20 14 -31 -25 38 36 2 -28 45 15 -38 -17 40 19 -41 -31 3 21 33 38 -20 40 48 25 -18 45 19 -21 -33 16 29 -42 9 -32 -27 26 -6 39 23 9 -5 34 -33 20 34 43 -43 9 -32 6 0 23 23 -1 37 41 11 23 -47 38 -22 -27 21 -34 18 37 0 47 36 1 38 -47 1 -10 22 4 36 40 -44 -39 -23 -23 22 3 -23 -16 25 41 -22 25 34 -34 -10 48 -22 30 13 -30 -19 -33 -10 -13 -37 