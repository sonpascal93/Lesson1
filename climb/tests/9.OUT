59
1 8 22 41 46 49 52 56 58 60 63 68 69 72 78 90 98 104 106 113 120 123 141 142 167 178 182 192 193 195 197 200 201 215 225 229 231 232 239 244 249 257 261 267 287 292 306 329 342 352 373 377 391 408 412 429 444 456 469 